code/go/0chain.net/blobbercore/reference/ref.go

```diff
@@ -422,6 +524,80 @@ func DeleteReference(ctx context.Context, refID int64, pathHash string) error {
 	return db.Where("path_hash = ?", pathHash).Delete(&Ref{ID: refID}).Error
 }
 
+func (r *Ref) SaveFile(ctx context.Context) error {
```

Comment:
@abouroumine Please rename this function to something like `SaveFileRef` or `SaveFileRow`. Also same for `SaveDir`